projects/packages/forms/src/blocks/field-telephone/deprecated.js

```diff
@@ -0,0 +1,11 @@
+import INNER_BLOCKS_DEPRECATION from '../shared/deprecations/inner-blocks-deprecation';
+
+export default [
+	{
+		...INNER_BLOCKS_DEPRECATION,
+		attributes: {
+			...INNER_BLOCKS_DEPRECATION.attributes,
+			label: { type: 'string', default: 'Phone' },
```

Comment:
(Nothing to action here): Tiny drive-by comment while I walk through the changes in the codebase: this `default: 'Phone'` does _not_ need a translate call because the original block did not translate this default label. Just thought I'd mention it in case anyone else runs into it.